docs/docs/guides/how-to-add-digest-to-email-notifications.md

```diff
@@ -0,0 +1,259 @@
+        e.preventDefault();
+        const response = await fetch("https://emaildigestbackend.onrender.com/api/v1/sending-email-digest", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify(formInput),
+
+        })
+        console.log(response.data);
+        setFormInput({ notif: '' })
+    }
+
+    const handleClick = () => {
+        setButtonClicked(true);
+
+        // Reset the button feedback after a certain duration
+        setTimeout(() => {
+            setButtonClicked(false);
+        }, 100);
+    };
+
+    const onChangeHandler = e => {
+        const value = e.target.name === 'email' ? e.target.value.trim() : e.target.value;
+
+        setFormInput((prev) => (
+            {
+                ...prev,
+                [e.target.name]: value
+            }
+        ))
+    }
+    return (
+            <div>
+            <h1 >Email Digest Playground</h1>
+            <p >Don't know how to? Start <a className="underline" href="https://docs.novu.co/platform/digest/"> here</a></p>
+            <form onSubmit={onSubmitHandler}>
+                <div>
+                    <div>
+                        <label htmlFor="email" className="w-48">
+                            Email:
+                        </label>
+                        <input
+                            placeholder="Your Email here"
+                            value={formInput.email}
+                            id="email"
+                            name="email"
+                            onChange={onChangeHandler}
+                        />
+                    </div>
+                    <div>
+                        <label htmlFor="notif">
+                            Notification:
+                        </label>
+                        <textarea
+                            placeholder="Enter the notification text"
+                            value={formInput.notif}
+                            id="notif"
+                            name="notif"
+                            onChange={onChangeHandler}
+                        />
+                    </div>
+                </div>
+
+                <button onClick={handleClick} type="submit">
+                    <BiRightArrowAlt className="text-2xl" />
+                </button>
+            </form>
+        </div>
+    )
+}
+```
+
+We're hitting the backend route we had created earlier. The backend has been deployed on render as can be seen in this code snippet above.
+
+Congratulations on following the guide up until this point. We can style our app a little using [some TailwindCSS](https://github.com/novuhq/digest-email-app).
+
+If you’ve done everything as recommended, you’ll end up with an app that looks like this:
+
+![email digest app showcase](https://res.cloudinary.com/dxc6bnman/image/upload/v1689098964/guides/Screenshot_2023-07-11_at_11.39.17_PM_xtbh3r.png)
```

Comment:
This should be a gif that shows the entry of the text and how it shows in your email box as a digest.

Reply:
added ✅